Sources/Examples/Globals.swift

```diff
@@ -0,0 +1,59 @@
+import Dispatch
+
+#if swift(<6.0)
+/// An unsafe global variable.
+///
+/// See swift-6-concurrency-migration-guide/commonproblems/#Sendable-Types
+var supportedStyleCount = 42
+#endif
+
+/// Version of `supportedStyleCount` that uses global-actor isolation.
+@MainActor
+var globallyIsolated_supportedStyleCount = 42
```

Comment:
nitpick but "globally isolated" is a term we don't have, perhaps just "`globalActorIsolated_...`"?

Reply:
I think consistent, correct terminology is really important!